sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/randomExpressions.scala

```diff
@@ -374,7 +374,7 @@ case class RandStr(
     var result: TypeCheckResult = TypeCheckResult.TypeCheckSuccess
     def requiredType = "INT or SMALLINT"
     Seq((length, "length", 0),
-      (seedExpression, "seedExpression", 1)).foreach {
+      (seedExpression, "seed", 1)).foreach {
```

Comment:
Since it is an id, could you quote it using to `toSQLId()`, please at:
```scala
                "inputName" -> toSQLId(name),
```

Reply:
Okay

Reply:
Given this, let me update `Uniform`'s `name` in this PR as well:
https://github.com/apache/spark/blob/861b5e98e6e4f61e376d756f085e0290e01fc8f4/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/randomExpressions.scala#L266